Mathlib/NumberTheory/ModularForms/JacobiTheta/TwoVariable.lean

```diff
@@ -0,0 +1,136 @@
+/-
+Copyright (c) 2023 David Loeffler. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Loeffler
+-/
+import Mathlib.Analysis.SpecialFunctions.Gaussian
+import Mathlib.Analysis.Complex.LocallyUniformLimit
+
+/-!
+# The two-variable Jacobi theta function
+
+This file defines the two-variable Jacobi theta function
+
+$$\theta(z, \tau) = \sum_{n \in \mathbb{Z}} \exp (2 i \pi n z + i \pi n ^ 2 \tau),$$
+
+and proves the functional equation relating the values at `(z, τ)` and `(z / τ, -1 / τ)`,
+using Poisson's summation formula. We also show holomorphy (in both variables individually,
+not jointly).
+-/
+
+open Complex Real Asymptotics Filter Topology
+
+open scoped Real BigOperators
+
+/-- The two-variable Jacobi theta function,
+`θ z τ = ∑' (n : ℤ), cexp (2 * π * I * n * z +  π * I * n ^ 2 * τ)`.
+
+The sum is only convergent for `0 < im τ`; we are implictly extending it by 0 for other values of
+`τ`. -/
+noncomputable def jacobiTheta₂ (z τ : ℂ) : ℂ :=
+  ∑' n : ℤ, cexp (2 * π * I * ↑n * z +  π * I * n ^ 2 * τ)
```

Comment:
There is an extra space after the `+`.Sorry,  I don't know how to do the comments that you can just click to add the changes.

Reply:
> Sorry, I don't know how to do the comments that you can just click to add the changes.

That's the first icon in the header above the comment box (looks like a sheet of paper with $\pm$ on it). It's most useful if you select a block of code at a time to comment on, rather than just one line, by dragging over the "+" signs at the start of the lines in the "Files changed" view.

Reply:
Ah thank you!